### ISSUE TITLE
Store: Theme dashboard widget

### ISSUE THREAD

**Issue by jameskoster** [Automattic/wp-calypso#16361]
There's a good chance that the theme users have active when they begin setting up their store is not going to be compatible with WooCommerce.

When that's the case, let's display a dashboard widget that explains the situation (in layman's terms) and suggests alternative themes to try.

Design to come.

**Comment by kellychoffman** [comment]
Closing this as we already have an issue for it: https://github.com/Automattic/wp-calypso/issues/16294
